llvm/docs/DeveloperPolicy.rst

```diff
@@ -539,11 +539,21 @@ What are the expectations around a revert?
 Obtaining Commit Access
 -----------------------
 
-We grant commit access to contributors that can provide a valid justification.
-If you would like commit access, please use this `link <https://github.com/llvm/llvm-project/issues/new?title=Request%20Commit%20Access%20For%20%3Cuser%3E&body=%23%23%23%20Why%20Are%20you%20requesting%20commit%20access%20?>`_ to file
```

Comment:
I'm a bit late to the party, but I would suggest something like the following introductory sentence:

> LLVM operates using Github Pull Requests. In most cases, contributing to the project does not require commit access, as Pull Request reviewers can perform the merge once ready. However, contributors are eligible to obtain direct commit access to the repository once they have 3 or more merged pull requests. To do so, you may use this link <...>

This clarifies that one can actually go a long way without requiring commit access.

Reply:
@ldionne I've submitted this as a follow up change here: https://github.com/llvm/llvm-project/pull/128244